docs/book/standards/packages-structure.md

```diff
@@ -1,25 +1,25 @@
 Packages Structure
 ===================
 
-A component must be treated as a simple PHP library. It means that must be 
+A component must be treated as a simple PHP library. This means that it must be
 framework agnostic. At the beginning, Elcodi was a set of Bundles, divided by
 concerns or boundaries, and implemented as a single symfony-framework Bundle.
 
-This was cause of different opinions, of course, like every single 
+This was because of different opinions, of course, like every single
```

Comment:
This was the root of disagreement with different opinions

This fact was the root cause of many disagreements.